packages/instant-apps-components/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.0.0-beta.299
+
+### instant-apps-scoreboard
+
+- fix: Only check for 'view.update' if 'queryType' = 'layerView'.
```

Comment:
I really need to follow your example and get better about updating the change log at both the app and component level